lib/routes/routeBackbeat.js

```diff
@@ -229,11 +230,28 @@ function multipleBackendPutObject(request, response, bucketInfo, objMd, log,
         });
 }
 
+function multipleBackendDeleteObject(request, response, bucketInfo, objMd,
+    log, callback) {
+    const storageLocation = request.headers['x-scal-storage-class'];
+    const objectGetInfo = {
+        key: request.objectKey,
+        dataStoreName: storageLocation,
+    };
+    const reqUids = log.getSerializedUids();
+    return multipleBackendGateway.delete(objectGetInfo, reqUids, err => {
+        if (err) {
```

Comment:
do you want to add a log at this layer after all?

Reply:
Updated.